README.md

```diff
@@ -218,6 +218,28 @@ AppID.getInstance().signinAnonymously(getApplicationContext(), new Authorization
 });
 ```
 
+## Sign in with refresh token
+It is recommended to store the refresh token locally such that it will be possible to sign in with the refresh token without requiring the user to type his credentials again.
+
```

Comment:
maybe needs to add 
** notice that user identity will not be updated if using the refresh token. To obtain the new identity please login and retrieve a new token.... or something like that

Reply:
I added this comment in the method itself (signinWithRefreshToken)